docs/ref/forms/fields.txt

```diff
@@ -13,6 +13,11 @@ When you create a ``Form`` class, the most important part is defining the
 fields of the form. Each field has custom validation logic, along with a few
 other hooks.
 
+.. note::
+   Some fields expect input in specific formats (e.g., dates, times, JSON).
+   See :ref:`form-field-formats` for guidelines on communicating these
+   requirements to users.
```

Comment:
I wouldn't put this here as I think if we want to cross reference we should add links in the help_text and error_messages as those are what we have solid recommendations for